airflow/providers/cncf/kubernetes/python_kubernetes_script.jinja2

```diff
@@ -17,7 +17,7 @@
  under the License.
 -#}
 
-import {{ pickling_library }}
+import json
```

Comment:
Why switching to JSON?

Reply:
Since we have to write out json (for K8s xcom output), I thought reading in json would be simpler. I’m realizing now though that users could pass in a dataclass, function, or something else that can’t be serialized via json so will switch back to pickle here.